### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by PetarPurac1**
Finish footer component. Ready for review.

### CODE REVIEW

File: zuko-ui/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const Footer = () => {
+  return (
+    <footer className={styles.footer}>
+        <div className={styles['footer-content']}>
+        <h3 className={styles['footer-title']}>Prijavite se na bilten</h3>
+        <form action="" className={styles['footer-form']}>
+            <label htmlFor=""></label>
+            <input type="email" id='' placeholder='Vaša mail adresa' className={styles['footer-input']}/>
+            <button className={styles['footer-btn']}>
+                <img src="/arrow-right.svg" alt="" />
```
Comment (mkovacevic01) [inline zuko-ui/src/components/Footer/Footer.jsx]: Add alt tag to images

File: zuko-ui/src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            </button>
+        </form>
+        <img src="/avioncic.png" alt="paper aeroplane img" className={styles['footer-img']} />
```
Comment (mkovacevic01) [inline zuko-ui/src/components/Footer/Footer.jsx]: We can use styles.footerImg convention, instead of kebap case "footer-img"

Comment (mkovacevic01) [reply zuko-ui/src/components/Footer/Footer.jsx]: Please use camel case convention for all classes

File: zuko-ui/src/components/Footer/Footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,72 @@
+.footer {
+    background-color: #FFF8E7;
+    padding: 121px 25px 70px;
+    @media only screen and (min-width: 1024px) {
```
Comment (mkovacevic01) [inline zuko-ui/src/components/Footer/Footer.module.scss]: Use our mixing for responsive. Use can see our test component to see how we import mixins in scss file

**Review by mkovacevic01 — CHANGES_REQUESTED** [review]

**Comment by PetarPurac1** [comment]
Unified class names to CamelCase convention, added missing alt tags to images. Added mixins for responsive design.
Ready for re-testing.

**Review by mkovacevic01 — APPROVED** [review]

**Pull request merged**